src/components/ViewFieldIngelogdExtraInfo/ViewFieldIngelogdExtraInfo.js

```diff
@@ -102,12 +113,26 @@ const ViewFieldIngelogdExtraInfo = ({ crudObject, hideEdit, className }) => {
     )
 }
 
+/**
+ * Component that renders the EigenarenList component, which displays the ownerlist.
+ *
+ * @component
+ *
+ * @param {object} eigenaren - Parameter containing a collection of owners.
+ */
 const EigenarenList = ({ eigenaren }) => {
     const getUsername = (item) => {
         if (!item) return null
         return item.Gebruikersnaam
     }
 
+    /**
+     * Function that gets the abbrevation from the username.
+     *
+     * @function
+     *
+     * @param {string} item - Parameter containing a text value.
```

Comment:
If you look at rule :126 you can see we access the .Gebruikersnaam property of the item that is passed. So this parameter is of type object.

Reply:
You forgot to update this one 😛